documentation/package.json

```diff
@@ -33,11 +33,12 @@
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "styled-components": "^5.1.1",
-    "three": "0.125.2"
+    "three": "0.128.0"
```

Comment:
![image](https://user-images.githubusercontent.com/4989157/120339675-97884b80-c2f5-11eb-8160-a9b4a126993f.png)

Three must be 0.125.2 not higher in order for reveal 1.x to work in 1.x version docs.

Reply:
Ok, I thought that part was sorted out - sorry. So I guess we need to wait until we have a solution of how to manage both 1 and version 2 docs?